spring-cloud-starter-stream-source-sftp-dataflow/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties

```diff
@@ -0,0 +1,27 @@
+configuration-properties.classes=\
+  org.springframework.cloud.stream.app.tasklaunchrequest.DataflowTaskLaunchRequestProperties,\
+  org.springframework.cloud.stream.app.trigger.TriggerPropertiesMaxMessagesDefaultUnlimited
+
+configuration-properties.names=\
+sftp.auto-create-local-dir,\
```

Comment:
Why the whole `SftpSourceProperties` class doesn't work for us here like those two mentioned above?

Reply:
Because two properties not listed do not apply here, stream mode and tasklaunchrequest

Reply:
OK. Can we extract them then to the separate properties class?
Although I may really live with this approach as well, if your find a new class as a bit awkward solution...

Reply:
Yeah extracting them to a separate class doesn't work with the boot metadata unless they have a unique prefix. I screwed around with this for 2 days.